src/Domain/CoordinateMaps/Composition.cpp

```diff
@@ -97,6 +109,61 @@ Composition<Frames, Dim, std::index_sequence<Is...>>::inv_jacobian(
   return inv_jacobian_impl(std::move(source_point), time, functions_of_time);
 }
 
+#ifdef SPECTRE_AUTODIFF
+template <typename Frames, size_t Dim, size_t... Is>
+InverseHessian<double, Dim, tmpl::front<Frames>, tmpl::back<Frames>>
+Composition<Frames, Dim, std::index_sequence<Is...>>::inv_hessian(
+    tnsr::I<double, Dim, SourceFrame> source_point, const double time,
+    const FuncOfTimeMap& functions_of_time) const {
+  if (supports_autodiff()) {
+    return inv_hessian_impl(std::move(source_point), time, functions_of_time);
+  } else {
+    ERROR("At least one of the Maps does not support autodiff");
```

Comment:
Is there a way to identify which map by name? I worry that we will hit this error and it'll be tricky to figure out what to change.